esp/esp/program/modules/handlers/resourcemodule.py

```diff
@@ -371,7 +371,7 @@ def resources(self, request, tl, one, two, module, extra, prog):
         if 'timeslot_form' not in context:
             context['timeslot_form'] = TimeslotForm()
         
-        context['resource_types'] = self.program.getResourceTypes().exclude(priority_default=0).order_by('priority_default')
+        context['resource_types'] = self.program.getResourceTypes().exclude(priority_default=0)
```

Comment:
I think the extra `.exclude()` may prevent this from actually using the cache, but I think that's okay in this case, since it's just a single query in an admin-facing view.